hoomd/md/angle.py

```diff
@@ -38,7 +38,7 @@ def _attach(self):
 class Harmonic(Angle):
     r"""Harmonic angle potential.
 
-    The command angle.harmonic specifies a harmonic potential energy between
+    :py:class:`Harmonic` specifies a harmonic potential energy between
```

Comment:
We might prefer to remove the Sphinx role? I was copying this from other classes and used an explicit role to match the examples I saw.
```suggestion
    `Harmonic` specifies a harmonic potential energy between
```

Reply:
Yes, we can remove Sphinx roles.